generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,528 @@
+You're not alone!
+
+For an approachable review of linear regression models, including a review of the equation behind them, check out this [article explaining linear regression](https://education.arcus.chop.edu/ordinary_linear_regression/).
+
+</div>
+
+For example, we could express a model [predicting heart rate from temperature in febrile children](https://www.frontiersin.org/articles/10.3389/fped.2020.548154/full) as the equation:
+
+$$
+\text{heart} \_ \text{rate} = \beta_0 + \beta_{\text{temp}} * \text{temp} + e
+$$ 
+
+If we wanted to look at differences in heart rate by sex (a categorical predictor) instead, the equation would actually look quite similar (assuming that the sex variable was coded 0 and 1 in the data): 
+
+$$
+\text{heart} \_ \text{rate} = \beta_0 + \beta_{\text{sex}} * \text{sex} + e
+$$ 
+
+What if you want to model sex and temperature together at the same time? No problem, you can add as many predictors to a linear model as you like: 
+
+$$
+\text{heart} \_ \text{rate} = \beta_0 + \beta_{\text{temp}} * \text{temp} + \beta_{\text{sex}} * \text{sex} + e
+$$ 
+
+And you can allow quite a lot of complexity and nuance in a linear model as well, by including things like interaction terms or polynomial terms. 
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+To learn more about interaction terms in linear models, checkout this [article about how to interpret interactions in a regression model](https://education.arcus.chop.edu/understanding-interactions/).
+
+</div>
+
+Linear models are so popular in statistics because equations for lines are computationally simple (even if they're not simple to understand!) but still allow us to model pretty complex relationships.
+
+It would be handy if linear models covered all possible research questions, but unfortunately they don't!
+**Generalized linear models are a way to extend the functionality of linear models to cover situations when they normally wouldn't work.**
+
+## When Ordinary Linear Models Fail
+
+Linear models are only appropriate when your outcome variable(s) are **continuous** and **unbounded** (we'll cover the definitions of continuity and boundedness more thoroughly after the following example). 
+If you want to model a categorical outcome, or one that's bounded (like a proportion or percent), then you can't rely on regular linear models to get the job done. 
+
+For example, let's return to the linear model example we were considering, predicting heart rate from temperature in febrile children. 
+As a reminder, here's the linear equation for that model: 
+
+$$
+\text{heart} \_ \text{rate} = \beta_0 + \beta_{\text{temp}} * \text{temp} + e
+$$ 
+
+But what if you're interested in predicting not heart rate, but whether or not the patient is at risk for sepsis, a [binary](https://en.wiktionary.org/wiki/binary) (BI-na-ree) outcome?
```

Comment:
The pronunciation I put in is closer to the one in the linked wiktionary page, so I'm guessing it might be more standard than bi-nair-ee (although I feel like that's closer to what I say, too). I think we should keep as-is.

Reply:
ok, fine by me!